documentation/modules/ROOT/pages/configuration/avro.adoc

```diff
@@ -9,57 +13,94 @@
+endif::product[]
 
-== Technical Information
+// Type: concept
+// Title: About the {registry}
+[id="about-the-registry"]
+== About the {registry-name-full}
 
-A system that wants to use Avro serialization needs to complete two steps:
+ifdef::community[]
+The link:https://github.com/Apicurio/apicurio-registry[{registry}] open-source project provides several components that work with Avro:
+endif::community[]
 
-* Deploy an https://github.com/Apicurio/apicurio-registry[Apicurio API/Schema Registry] instance
-* Install the Avro converter from https://repo1.maven.org/maven2/io/apicurio/apicurio-registry-distro-connect-converter/{apicurio-version}/apicurio-registry-distro-connect-converter-{apicurio-version}-converter.tar.gz[the installation package] into Kafka's _libs_ directory or directly into a plug-in directory
-* Use thes the following properties to configure Apache Connect instance
+ifdef::product[]
+{LinkServiceRegistryGetStart}[{registry-name-full}] provides several components that work with Avro:
+endif::product[]
 
-[source]
+* An Avro converter that you can specify in {prodname} connector configurations. This converter maps Kafka Connect schemas to Avro schemas. The converter then uses the Avro schemas to serialize the record keys and values into Avro's compact binary form.
+
+* An API and schema registry that tracks:
++
+** Avro schemas that are used in Kafka topics
+** Where the Avro converter sends the generated Avro schemas
+
++
+Since the Avro schemas are stored in this registry, each record needs to contain only a tiny _schema identifier_.
+This makes each record even smaller. For an I/O bound system like Kafka, this means more total throughput for producers and consumers.
+
+* Avro _Serdes_ (serializers and deserializers) for Kafka producers and consumers. Kafka consumer applications that you write to consume change event records can use Avro Serdes to deserialize the change event records.
+
+To use the {registry} with {prodname}, add {registry} converters and their dependencies to the Kafka Connect container image that you are using for running a {prodname} connector.
+
+[NOTE]
+====
+The {registry} project also provides a JSON converter. This converter combines the advantage of less verbose messages with human-readable JSON. Messages do not contain the schema information themselves, but only a schema ID.
+====
+
+// Type: concept
+// Title: Overview of deploying a {prodname} connector that uses Avro serialization
+[id="overview-of-deploying-a-debezium-connector-that-uses-avro-serialization"]
+== Deployment overview
+
+To deploy a {prodname} connector that uses Avro serialization, there are three main tasks: 
+
+ifdef::community[]
+. Deploy an link:https://github.com/Apicurio/apicurio-registry[{registry-name-full}] instance.
+endif::community[]
+ifdef::product[]
+. Deploy a link:{LinkServiceRegistryGetStart}[{registry-name-full} instance by following the instructions in {NameServiceRegistryGetStart}].
```

Comment:
This doesn't render (yet?) into working link

Reply:
Right. This doc has not yet been published. It should not work yet.